### ISSUE TITLE
[Suggestion] Teleport to plot UUID (with permission node)

### ISSUE THREAD

**Issue by akiraic** [LlmDl/Towny#2561]
Allow players to teleport to named plots. Let's say I use _/plot set name freeplot1_. I want players to teleport to that freeplot1. 
Or put this function under /resident, so players can rename their own residences and others are now able to teleport to it. 
Or neither: just make possible for players to teleport to resident plot using _/res tp <playername>_, and it would put then right outside the said plot (if multiple, tp to the first plot bought by the said player).

If a plot would receive a number ID on the creation (/town claim), it would already make it easy to teleport to it.

## **In short, we need some sort of plot/res "address" system.**

**Comment by LlmDl** [comment]
While this is an interesting idea, it won't work because multiple plots can have the same name. Many people use names to apply names to entire roadways, or large buildings which span mulitple plots. 

There could be a system for teleporting to townblock coordinates in your town but that that point it could be accomplished much easier with a custom plugin, as many Towny servers wouldn't allow a mayor/resident to teleport to any plot in their town at will.

**Comment by akiraic** [comment]
But that is why i put there a line about "town block ID", that would give a number address to the plot on creation. That would be the UUID of the plot, allowing for different plots having the same name, but still have the possibility to teleport to each one individually. And the ID would be shown in the plot info, or even with a different command, like /plot address or something.

**Comment by LlmDl** [comment]
As of now there's an admin command for teleporting to towny chunks, added in 0.92.0.2.

/ta tpplot {world} {x} {z}

If this doesn't satisfy your needs please ask to have this ticket reopened.
